src/CharCreation/Attributes.elm

```diff
@@ -89,6 +92,17 @@ progressBarStyle val =
         , ( "min-width", "0" )
         ]
 
+tickStyle: Int -> Html.Attribute Msg
+tickStyle val =
+    style
+        [ ( "width", (toString val) ++ "%" )
```

Comment:
ah ok, i didn't know that. but i like the parens too, it removes ambiguity for those who don't know.

I haven't done much in the way of styling after the landing page and this char creation page, it stopped my flow too much learning semantic and going back to elm. so i've left it all to the end.

I'd like to use semantic as much as possible too, rather than doing custom css. that way, it keeps the styles consistent and working on all widths.